components/link/Link.js

```diff
@@ -0,0 +1,54 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import s from './theme.css';
+
+class Link extends PureComponent {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    external: PropTypes.bool,
+    onClick: PropTypes.func,
+    url: PropTypes.string,
+  };
+
+  static defaultProps = {
+    className: '',
+    external: false,
```

Comment:
I know I commented on previous PRs that having falsey values as `defaultProps` is kinda redundant, but I'm having second thoughts about it now.

WDYT @maartenscholz @lowiebenoot @svenlombaert ?

Reply:
I don't mind it, especially for optional boolean props

Reply:
Not really strong opinionated about it. I don't mind having them, but i also don't mind not having them.